cardano-config/src/Cardano/Config/LedgerQueries.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE FlexibleInstances #-}
+
+module Cardano.Config.LedgerQueries
```

Comment:
We're trying to eliminate `cardano-config`, can you move this to `cardano-node`?

Reply:
I need `LedgerQueries` in `TraceConstraints`, so, unfortunately, I don't think so..

Reply:
Hmm...alright, I'll need to move these later.